app/src/main/java/com/dailyaims/screens/addtask/AddTaskFragment.kt

```diff
@@ -32,6 +32,8 @@ class AddTaskFragment : Fragment() {
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
+       val aimsModel= arguments?.getSerializable("Aims") as AimsModel?
```

Comment:
replace cast to null type with null safe cast
"as AimsModel?" to "as? AimsModel"